docs/source/markdown/podman-create.1.md

```diff
@@ -364,6 +364,13 @@ The following example maps uids 0-2000 in the container to the uids 30000-31999
 
 Add additional groups to run as
 
+#### **--groups**=*false*
+
+Allow container to use the user's supplementary group access. If file systems or
+devices are only accessible by the rootless users group, this flag tells the OCI
```

Comment:
"This ensures that the permissions of the container's root user will match that of the user launching Podman, ensuring that the container will be able to access files or devices the user can access via a supplemental group"